packages/imperative/src/rest/src/session/ConnectionPropsForSessCfg.ts

```diff
@@ -405,10 +406,33 @@ export class ConnectionPropsForSessCfg {
             const answers: { [key: string]: any } = {};
             const profileSchema = this.loadSchemaForSessCfgProps(connOpts.parms, promptForValues);
             const serviceDescription = connOpts.serviceDescription || "your service";
+            let tokenInstructions: string = "";
+            let somePromptHasOccurred = false;
 
             for (const propNm of promptForValues) {
                 const sessPropNm = AuthOrder.getPropNmFor(propNm, PropUse.IN_SESS);
                 const cfgPropNm = AuthOrder.getPropNmFor(propNm, PropUse.IN_CFG);
+                if ((propNm === "tokenType" || propNm === "tokenValue") && connOpts.parms?.response.console.log) {
+                    // Typically users will not know how to type in a token.
+                    // Add extra instructions for obtaining a token and only display them once.
+                    if (tokenInstructions.length === 0) {
+                        // instructions have not yet been displayed
+                        if (somePromptHasOccurred) {
```

Comment:
(nitpick) `somePromptHasOccurred` is technically redundant since you can infer whether a prompt has occurred based on whether `answers` has at least one key. But, I think its fine to keep it for readability and as an explicit flag.

Reply:
Since I made other changes, I removed somePromptHasOccurred in favor of "answers" to reduce clutter.